shared/networking/2.0.md

```diff
@@ -0,0 +1,202 @@
+# Networking on resinOS 2.0
```

Comment:
As far as I understand doxx partials this will only be selected when selecting `resinos 2.0` this looks like more general info that should be pulled in for 2.x filter. If this is the case it's something we should look at later.